### ISSUE TITLE
[BUG] CRC start fail on vSphere CentOS 8 VM

### ISSUE THREAD

**Issue by Les-Linux** [code-ready/crc#1249]
### General information

  * OS: Linux
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: VM - CentOS8 on vsphere 6.7u3

## CRC version
```bash
# crc version: 1.10.0+9025021
```
  
## CRC status
```bash
# ERRO stat /home/cheServiceAccount/.crc/machines/crc/id_rsa: no such file or directory
```

## CRC config
```bash
# - memory                                : 16384
- pull-secret-file                      : /tmp/pull-secret.txt
```

## Host Operating System
```bash
# NAME="CentOS Linux"
VERSION="8 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:8"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-8"
CENTOS_MANTISBT_PROJECT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="8"
```

### Steps to reproduce

  1. Deploy a new instance of CentOs 8 on vsphere 6.7u3
  2. Ensure  Expose hardware assisted virtualization to the guest OS is enabled
  3. Ensure Enable virtualized CPU performance counters is enabled
  4. run crc setup
  5. run crc start

### Expected
Successful start of openshift cluster

### Actual
Error creating host: Error creating the VM: Error creating machine: Error in driver during machine creation: read tcp 127.0.0.1:57590->127.0.0.1:38571: read: connection reset by peer

### Logs
https://gist.github.com/Les-Linux/803d5cb5142ca0171bc8536ca8d8e214

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```
# Tried all the above

Gist - Debug output https://gist.github.com/Les-Linux/803d5cb5142ca0171bc8536ca8d8e214

**Comment by Les-Linux** [comment]
Is there anything I am missing in order to get this up and running?

**Comment by cfergeau** [comment]
This line in the logs is unexpected, not sure what it means..
```
Wrapper Docker Machine process exiting due to closed plugin server (read tcp 127.0.0.1:35966->127.0.0.1:43681: read: connection reset by peer)
```
After a failed `crc start`, is the `crc` VM running? You can check with `virsh -c qemu:///system list --all` or `ps aux |grep qemu`. If it's running, is a ssh connection possible (`ssh api.crc.testing` or `ssh 192.168.122.11`)?

**Comment by gbraad** [comment]
I might be that the VM already had crashed ? Can you be sure the VM is operational... you could try to connect to the console of the VM and inspect the startup messages.
